GDAPI/GDAPI/Objects/GeometryDash/LevelObjects/GeneralObject.cs

```diff
@@ -43,9 +45,11 @@ static GeneralObject()
 
             objectTypes = typeof(GeneralObject).Assembly.GetTypes().Where(t => typeof(GeneralObject).IsAssignableFrom(t)).ToArray();
             objectTypeDictionary = new ObjectTypeInfoDictionary();
-            foreach (var t in objectTypes.Select(t => ObjectTypeInfo.GetInfo(t)).ToArray())
+            foreach (var t in objectTypes.Select(t => ObjectTypeInfo.GetInfo(t)))
                 if (t != null)
-                    objectTypeDictionary.Add(t);
+                {
```

Comment:
any particular reason why we're using a body here?

Reply:
An oversight, no body is intended to be used